src/main/java/codeu/controller/RegisterServlet.java

```diff
@@ -0,0 +1,19 @@
+package codeu.controller;
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/**
+ * Servlet class responsible for user registration.
+ */
+public class RegisterServlet extends HttpServlet {
+
+ @Override
+     public void doGet(HttpServletRequest request, HttpServletResponse response)
+     throws IOException, ServletException {
+
+     request.getRequestDispatcher("/WEB-INF/view/register.jsp").forward(request, response);
+ }
+}
```

Comment:
Remove the character after the closing } on line # 19

Reply:
Done!